index.js

```diff
@@ -66,9 +86,65 @@ export default class VIForegroundService {
      *
      * @return Promise
      */
-    static async stopService() {
+    async stopService() {
```

Comment:
since we change public API, we need to update README (and references) with these changes